basalt/nn/layers/conv.mojo

```diff
@@ -8,7 +8,7 @@ from basalt.autograd.attributes import AttributeVector, Attribute
 
 @always_inline("nodebug")
 fn q_sqrt(value: Float32) -> Float32:
-    var y = bitcast[DType.float32](0x5f3759df - (bitcast[DType.uint32](value) >> 1))
+    var y = bitcast[DType.float32](0x5F3759DF - (bitcast[DType.uint32](value) >> 1))
     return y * (1.5 - 0.5 * value * y * y)
```

Comment:
Since we are using this exact formula twice in two different files, we should probably extract it to a different file, can take care of when im home.